components/style/servo/selector_parser.rs

```diff
@@ -221,41 +221,9 @@ impl PseudoElement {
         }
     }
 
-    /// For most (but not all) anon-boxes, we inherit all values from the
-    /// parent, this is the hook in the style system to allow this.
-    ///
-    /// FIXME(emilio): It's likely that this is broken in a variety of
-    /// situations, and what it really wants is just inherit some reset
-    /// properties...  Also, I guess it just could do all: inherit on the
-    /// stylesheet, though chances are that'd be kinda slow if we don't cache
-    /// them...
+    /// To be removed.
```

Comment:
Why do we keep it then?

Reply:
Because I'm removing it in that Gecko bug I point above, which I'll sink in a few..

Reply:
sync*